css/src/components/form/radio.scss

```diff
@@ -0,0 +1,117 @@
+$radio-circle-size: 1.25em !default;
+$radio-dot-size: 0.625em !default;
+$radio-dot-checked-color: $primary !default;
+$radio-dot-color: $text-subtle !default;
+$radio-timing-function: $input-timing-function !default;
+$radio-duration: $input-transition-duration !default;
+$radio-animation: boop !default;
+$radios-spacing: 0.5em !default;
+
+@mixin radio-dot {
+	position: absolute;
+	inset: 0;
+	width: $radio-dot-size;
+	height: $radio-dot-size;
+	margin: auto;
+	border-radius: 50%;
+	background-color: $body-background;
+	content: '';
+	overflow: hidden;
+}
+
+@mixin radio-dot-checked {
+	background: $radio-dot-checked-color;
+}
+
+.radio {
+	display: inline-flex;
+	position: relative;
+	align-items: flex-start;
+	line-height: 1.25;
+	cursor: pointer;
+
+	.radio-dot {
+		display: inline-block;
+		position: relative;
+		inset-block-start: 0.0625em;
+		flex-shrink: 0;
+		width: $radio-circle-size;
+		height: $radio-circle-size;
+		border: 1px solid $text;
+		border-radius: $radius-rounded;
+		color: $radio-dot-color;
+		appearance: none;
+
+		&::before {
+			@include radio-dot;
+		}
+	}
+
+	.radio-label-text {
+		margin: 0 $radios-spacing;
+	}
+
+	legend + & {
```

Comment:
I think in design the word "Label" in the screenshot below was incorrectly used, more likely it matched the legend. So the question is should `legend` be styled like that? 

<img width="237" alt="image" src="https://user-images.githubusercontent.com/57374379/158251599-4c5bbc49-d445-441a-85f2-68bdac40d49a.png">

Reply:
I think we should remove this entirely if there are no major repercussions for updates in docs-ui

Reply:
Removed, but it means we'll need to add `margin-top-xxs` in numerous places where we use the fieldset.

Reply:
That's what I've done for a bunch of `labels` after migrating to docs 😁